### DISCUSSION TITLE
Can I add space between stacked bar charts ?

### DISCUSSION THREAD

**Discussion by VigneshTrantor** [airbnb/visx#1185] [Q&A] [answered]
Hey guys, 
can I add space between stacked bar charts ?
![stacked](https://user-images.githubusercontent.com/73022493/115866244-09e45100-a457-11eb-8bc5-013c28b47fa5.png)

I want it to have space similar to the one given in XY charts docs example like this
![barStacckwithspace](https://user-images.githubusercontent.com/73022493/115866364-326c4b00-a457-11eb-9f09-cc2b0dcab955.png)

Here is a codesandbox for your reference https://codesandbox.io/s/barcharts-stacked-with-space-try-zws5m?file=/Example.tsx

**Comment by maximerivolet** [comment, marked as answer]
Hi @VigneshTrantor,

There is an explanation here : https://github.com/airbnb/visx/discussions/1000

Especially this picture : 

https://user-images.githubusercontent.com/4496521/104070198-2cf63800-51bb-11eb-8f0a-be1558c19d51.png

Is this okay for you ?

**Reply by VigneshTrantor** [reply]
Thank you @goldenivan

**Reply by VigneshTrantor** [reply]
@goldenivan 
Is there some way to avoid touching the vertical chart lines ?
![touching](https://user-images.githubusercontent.com/73022493/116963815-82ed6f00-acc7-11eb-917a-f1c333615578.png)

I want the bars to be neatly aligned in the center like this:
![no-touching](https://user-images.githubusercontent.com/73022493/116963836-926cb800-acc7-11eb-96a6-b4be8dadf46e.png)

In my codesandbox, the grid lines are coming in the center for some reason https://codesandbox.io/s/barcharts-stacked-with-space-try-zws5m?file=/Example.tsx

**Reply by maximerivolet** [reply]
Hi, maybe try to adjust columns of the grid.
